orchagent/mirrororch.cpp

```diff
@@ -758,72 +785,56 @@ void MirrorOrch::updateFdb(const FdbUpdate& update)
 void MirrorOrch::updateLagMember(const LagMemberUpdate& update)
 {
     SWSS_LOG_ENTER();
-    auto sessionIter = m_syncdMirrors.begin();
-    for (; sessionIter != m_syncdMirrors.end(); ++sessionIter)
-    {
-        if (!sessionIter->second.neighborInfo.resolved)
-        {
-            continue;
-        }
 
-        if (sessionIter->second.neighborInfo.port.m_type != Port::LAG)
-        {
-            continue;
-        }
+    for (auto it = m_syncdMirrors.begin(); it != m_syncdMirrors.end(); it++)
+    {
+        const auto& name = it->first;
+        auto& session = it->second;
 
-        // It is possible to have few session that points to one LAG member
-        if (sessionIter->second.neighborInfo.port != update.lag || sessionIter->second.neighborInfo.portId != update.member.m_port_id)
+        // Check the following two conditions:
+        // 1) the neighbor is LAG
+        // 2) the neighbor LAG matches the update LAG
+        if (session.neighborInfo.port.m_type != Port::LAG ||
```

Comment:
is this port LAG or LAG_MEMBER?

Reply:
LAG. The neighbor will be the router interface. So it cannot be a LAG_MEMBER.